fe/fe-core/src/main/java/com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java

```diff
@@ -263,6 +266,25 @@ public Void visitDropMaterializedViewStatement(DropMaterializedViewStmt stmt, Co
             stmt.getDbMvName().normalization(context);
             return null;
         }
+
+        @Override
+        public Void visitAlterMaterializedViewStatement(AlterMaterializedViewStatement statement,
+                                                        ConnectContext context) {
+            statement.getMvName().normalization(context);
+            if (statement.getNewMvName() != null) {
+                String newTableName = statement.getNewMvName();
+                if (statement.getMvName().getTbl().equalsIgnoreCase(newTableName)) {
+                    throw new SemanticException("New table name can not equals to old name");
+                }
+            }
+            final RefreshSchemeDesc refreshSchemeDesc = statement.getRefreshSchemeDesc();
```

Comment:
here should support modify time interval? it seems that RefreshSchemeDesc do not contain interval field